cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -110,7 +110,9 @@ func init() {
 
 	describeClusterClusterCmd.Flags().BoolVar(&dc.disableNoEcho, "disable-no-echo", false, ""+
 		"Disable hiding of a MachineInfrastructure and BootstrapConfig when ready condition is true or it has the Status, Severity and Reason of the machine's object.")
-	describeClusterClusterCmd.Flags().BoolVar(&dc.disableGrouping, "disable-grouping", false,
+	describeClusterClusterCmd.Flags().BoolVar(&dc.grouping, "grouping", true,
+		"Groups machines when ready condition has the same Status, Severity and Reason.")
+	describeClusterClusterCmd.Flags().BoolVar(&dc.grouping, "disable-grouping", false,
```

Comment:
You can't have two flags using the same variable
```suggestion
	describeClusterClusterCmd.Flags().BoolVar(&dc.disableGrouping, "disable-grouping", false,
```
Also the flag should be deprecated as per https://pkg.go.dev/github.com/spf13/pflag#readme-deprecating-a-flag-or-its-shorthand

Reply:
yes marked it as deprecated